src/Pim/Bundle/CatalogBundle/Resources/config/serializers_indexing.yml

```diff
@@ -0,0 +1,82 @@
+parameters:
+    pim_catalog.normalizer.indexing.product.class: Pim\Component\Catalog\Normalizer\Indexing\ProductNormalizer
+    pim_catalog.normalizer.indexing.product.properties.class: Pim\Component\Catalog\Normalizer\Indexing\Product\PropertiesNormalizer
+    pim_catalog.normalizer.indexing.product.associations.class: Pim\Component\Catalog\Normalizer\Indexing\Product\AssociationsNormalizer
+    pim_catalog.normalizer.indexing.product.product_values.class: Pim\Component\Catalog\Normalizer\Indexing\Product\ProductValuesNormalizer
+    pim_catalog.normalizer.indexing.product.product_value.class: Pim\Component\Catalog\Normalizer\Indexing\Product\ProductValueNormalizer
+    pim_catalog.normalizer.indexing.product.price.class: Pim\Component\Catalog\Normalizer\Indexing\Product\PriceNormalizer
+    pim_catalog.normalizer.indexing.product.metric.class: Pim\Component\Catalog\Normalizer\Indexing\Product\MetricNormalizer
+    pim_catalog.normalizer.indexing.datetime.class: Pim\Component\Catalog\Normalizer\Indexing\DateTimeNormalizer
+    pim_catalog.normalizer.indexing.file.class: Pim\Component\Catalog\Normalizer\Indexing\FileNormalizer
+    pim_catalog.normalizer.indexing.attribute_option.class: Pim\Component\Catalog\Normalizer\Indexing\AttributeOptionNormalizer
+services:
+    pim_catalog.normalizer.indexing.product:
+        class: '%pim_catalog.normalizer.indexing.product.class%'
+        arguments:
+            - '@pim_catalog.normalizer.indexing.product.properties'
+            - '@pim_catalog.normalizer.indexing.product.associations'
+        tags:
+            - { name: pim_serializer.normalizer, priority: 90 }
```

Comment:
Shouldn't we have a dedicated normalizer instead on relying on the generic (and over-used) pim_serializer ?